cloudwatch_logs_common/src/utils/log_file_manager.cpp

```diff
@@ -76,8 +76,8 @@ FileObject<LogCollection> LogFileManager::readBatch(
   }
 
   if(batch_size != log_data.size()){
-    AWS_LOG_INFO(__func__, "%d logs were not batched since the time"
-      " difference was > 24 hours. Will try again in a separate batch./n"
+    AWS_LOG_WARN(__func__, "%d logs were not batched since the time"
```

Comment:
Generally it is good practice to have a PR do one thing; makes reverting easier. This looks like something that can be easily merged in another PR. But that is just my opinion, I don't know if its a big deal.

Reply:
I think this is fine (small enough PR as well).